Makefile

```diff
@@ -101,3 +125,26 @@ release: test dist
 	git push --tags
 	open https://github.com/$(NAME)/releases/tag/v$(VERSION)
 	open dist
+
+.PHONY: generate-grpc
+generate-grpc:
+	$(PROTOC) -I=./grpc --gogo_out=\
+	Mgoogle/api/annotations.proto=github.com/gogo/googleapis/google/api,\
+	Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,plugins=grpc:./grpc \
+	--grpc-gateway_out=\
+	Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/protobuf/empty.proto=github.com/gogo/protobuf/types,\
+	Mgoogle/api/annotations.proto=github.com/gogo/googleapis/google/api,\
+	Mgoogle/protobuf/field_mask.proto=github.com/gogo/protobuf/types:./grpc \
+	grpc/authn.proto grpc/authn-private.proto
+	# Workaround for https://github.com/grpc-ecosystem/grpc-gateway/issues/229
+	sed -i.bak "s/empty.Empty/types.Empty/g" grpc/authn.pb.gw.go && rm grpc/authn.pb.gw.go.bak
```

Comment:
We used to have `Empty` during the young life of this PR, then they were removed in favor of our own empty messages. I left this here in case `Empty` shows up in the future for whatever reason and we need this hack.